cmd/thanos/receive.go

```diff
@@ -88,6 +88,8 @@ func registerReceive(app *extkingpin.App) {
 			EnableExemplarStorage:          true,
 			HeadChunksWriteQueueSize:       int(conf.tsdbWriteQueueSize),
 			EnableMemorySnapshotOnShutdown: conf.tsdbMemorySnapshotOnShutdown,
+			//TODO (rabenhorst): Do we want to make this configurable?
+			EnableNativeHistograms: true,
```

Comment:
Should we put this behind a flag?

Reply:
We can put it behind a hidden flag until we have full support.